numpy/core/src/private/lowlevel_strided_loops.h

```diff
@@ -414,20 +414,23 @@ PyArray_PrepareThreeRawArrayIter(int ndim, npy_intp *shape,
                             char **out_dataC, npy_intp *out_stridesC);
 
 /*
- * Return number of elements that must be peeled from
- * the start of 'addr' with 'nvals' elements of size 'esize'
- * in order to reach 'alignment'.
- * alignment must be a power of two.
- * see npy_blocked_end for an example
+ * Return number of elements that must be peeled from the start of 'addr' with
+ * 'nvals' elements of size 'esize' in order to reach 'alignment'.  alignment
+ * must be a power of two.  see npy_blocked_end for an example
```

Comment:
This comment seems to no longer be correct, although I realize it wasn't introduced in this patch.

Reply:
It is correct, the offset is obtained by using the lower order bits of the address. I was actually thinking of using the twos complement check for zero or power of two of unsigned integers, `(a & -a) == a`

Reply:
I mean that `npy_blocked_end ` doesn't seem to be an example of this function any more.

Reply:
Ah, it meant the example is in the `npy_blocked_end` function documentation. I have clarified the comment.